charts/flyte-core/values.yaml

```
@@ -15,7 +15,7 @@ flyteadmin:
   replicaCount: 1
   image:
     # -- Docker image for Flyteadmin deployment
-    repository: cr.flyte.org/flyteorg/flyteadmin
+    repository: cr.flyte.org/flyteorg/flyteadmin # FLYTEADMIN_IMAGE
```

Shouldn't these change to be
```suggestion
    repository: cr.flyte.org/flyteorg/flyteadmin-release # FLYTEADMIN_IMAGE
```
And the same for the others?

---

No that is not required, We only prepare them for release.

---

@EngHabu it's ready now